specs/experimental/builder.md

```diff
@@ -0,0 +1,172 @@
+    Note right of Builder: timespan for building blocks
+    OPS->> BB: builder_getPayloadV1
+    BB-->>OPS: BuilderPayloadV1
+    OPS-->>OPS: SimulatePayload
+    OPS-->>OPS: ConfirmPaylaod
+    OPS ->> EES: engine_forkchoiceUpdatedV3
+
+```
+
+- **Fork Choice Update**: The Sequencer propagates a Fork Choice Update to the Block Builder, indicating an update
+to the chain's latest head.
+- **Forward Transaction**: The Sequencer's Execution Engine forwards transactions it received to the Block Builder
+to be included in a block. *This step is not necessary if the Block Builder can get transactions elsewise and forwarding
+does not give any specific inclusion gurantees.*
+- **Get Payload**: The Sequencer requests a block from the Block Builder for a specific head.
+
+### Requesting a Block
+
+The block request mechanism ***MUST*** be triggered when the Driver schedules a new Fork Choice Update on the
+Sequencer. The Sequencer will translate the received Payload Attributes into a Payload Request for the
+Block Builder. As specified lower, the Sequencer ***MUST*** simulate the received payload to ensure correctness until
+an accountability mechanism can be introdcued.
+
+#### Liveness Failsafe
+
+To maintain network liveness while utilizing the Builder API, the Sequencer ***MUST*** operate an auxiliary process
+when building blocks. This process concurrently executes a Builder API request to the Block Builder alongside a local
+block production request through its local execution engine. This two-pronged strategy for generating blocks ensures
+that network liveness persists, even in instances where the Block Builder's block construction process experiences
+delays or is offline. This fallback mechanism should be seen as a training wheel.
+
+### Mempool Forwarding
+
+A builder network's throughput is conditional on the transactions it sees. Thus the Sequencer's Execution Engine,
+or simply it's RPC, can forward transactions to the Builder as part of regular mempool management, ensuring that
+user transactions are included in the Block Builder's block construction process efficiently.
+
+### Builder Configuration
+
+A builder is defined as the tuple (`builderPubkey`, `builderUrl`). The Sequencer is responsible for managing this
+tuple, but it will eventually live on the
+L1 [`SystemConfig`](https://github.com/ethereum-optimism/specs/blob/main/specs/protocol/system_config.md)
+where changes are emitted as an event. ***Builder's have no restriction or policies enforced on them at this time.***
+
+## Structures
+
+### `PayloadRequestV1`
+
+This structure contains information necessary to request a block from a local Block Builder.
+
+- `blockNumber`: `uint256`
+- `parentHash`:  `Hash32`
+- `pubKey`: `Address`
+- `gasLimit`: `uint256`
+- `parentBeaconBlockRoot`: `Hash32`
+
+### `BuilderPayloadV1`
+
+This structure represents the Block Builder's response to the request for payload.
+
+- `executionPayload`: `ExecutionPayloadV2`
```

Comment:
should this be updated to [`ExecutionPayloadV3`](https://github.com/ethereum/execution-apis/blob/main/src/engine/cancun.md#executionpayloadv3)?